Signal.xcodeproj/project.pbxproj

```diff
@@ -3121,13 +3138,13 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
I didn't modify this line, but it's notable: it appears we can target a different minimum version of iOS for our share extension. Score.